<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>task_one</class>
 <widget class="QWidget" name="task_one">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(248, 255, 243);</string>
  </property>
  <widget class="QLabel" name="label_task">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font: 300 9pt &quot;Yu Gothic UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>340</y>
     <width>141</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_return">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>240</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(39, 39, 39);
color: rgb(255, 255, 255);
font: 300 9pt &quot;Yu Gothic UI Light&quot;;</string>
   </property>
   <property name="text">
    <string>Return to menu</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>351</width>
     <height>191</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="1">
       <widget class="QLineEdit" name="lineEditMessage">
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
	border: 1px solid rgb(39,39,39);
	border-radius: 550px;
	background-color: rgb(255,255,255);
}
</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="labelFilePath">
        <property name="text">
         <string>File Path</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="labelMessage">
        <property name="text">
         <string>Message</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="lineEditFilePath">
        <property name="styleSheet">
         <string notr="true">QLineEdit
{
	border: 1px solid rgb(39,39,39);
	border-radius: 550px;
	background-color: rgb(255,255,255);
}
</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="pushButton_send">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(39, 39, 39);
color: rgb(255, 255, 255);
font: 300 9pt &quot;Yu Gothic UI Light&quot;;</string>
        </property>
        <property name="text">
         <string>Send</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="labelMessageServ">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>310</y>
     <width>411</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
